[nnc] Don't fuse fp16 on CPU

bertmaher opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#56119 [nnc] Don't fuse fp16 on CPU**
* #56118 [nnc][trivial] Trailing underscore style for llvmCode, asmCode members
* #56117 [nnc] Separate printing of optimized llvm bitcode from assembly

There are apparently still more issues with fp16 on LLVM so let's just
nuke it from orbit while we develop a robust workaround.

Differential Revision: [D27787080](https://our.internmc.facebook.com/intern/diff/D27787080/)

bertmaher committed:
[nnc] Don't fuse fp16 on CPU

There are apparently still more issues with fp16 on LLVM so let's just
nuke it from orbit while we develop a robust workaround.

Differential Revision: [D27787080](https://our.internmc.facebook.com/intern/diff/D27787080/)

[ghstack-poisoned]

bertmaher committed:
format on "[nnc] Don't fuse fp16 on CPU"


There are apparently still more issues with fp16 on LLVM so let's just
nuke it from orbit while we develop a robust workaround.

Differential Revision: [D27787080](https://our.internmc.facebook.com/intern/diff/D27787080/)

[ghstack-poisoned]

ZolotukhinM reviewed (approved):
Sounds good to me!

bertmaher committed:
Update on "[nnc] Don't fuse fp16 on CPU"


There are apparently still more issues with fp16 on LLVM so let's just
nuke it from orbit while we develop a robust workaround.

Differential Revision: [D27787080](https://our.internmc.facebook.com/intern/diff/D27787080/)

[ghstack-poisoned]

walterddr reviewed (commented):
could you please fix flake8 and reimport?

bertmaher committed:
flake8 on "[nnc] Don't fuse fp16 on CPU"


There are apparently still more issues with fp16 on LLVM so let's just
nuke it from orbit while we develop a robust workaround.

Differential Revision: [D27787080](https://our.internmc.facebook.com/intern/diff/D27787080/)

[ghstack-poisoned]
